### ISSUE TITLE
Being Picky about the Wiki

### ISSUE THREAD

**Issue by ReginaTL** [ritjoe/hfoss#490]
Currently there are no headers for the wiki, I feel it would add some clarity to put things under header such as Assignments, Syllabus/Admininstration, and Lectures.

**Comment by ritjoe** [comment]
Yes, I have aspirations for making better use of the the wiki's home page for this purpose, so thanks for the prompt and for the request.
